patches/server/0091-Disable-Vanilla-Chunk-GC.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Aikar <aikar@aikar.co>
+Date: Sun, 4 Dec 2022 12:48:52 -0300
+Subject: [PATCH] Disable Vanilla Chunk GC
+
+Bukkit has its own system for this.
+
+diff --git a/src/main/java/net/minecraft/server/WorldServer.java b/src/main/java/net/minecraft/server/WorldServer.java
+index 0700b5aec26a6a4ba4db4d3c66b8b15c295c5a03..ab107574c8027debefb437f1605789b1d48d5593 100644
+--- a/src/main/java/net/minecraft/server/WorldServer.java
++++ b/src/main/java/net/minecraft/server/WorldServer.java
+@@ -964,6 +964,7 @@ public class WorldServer extends World implements IAsyncTaskHandler {
+             this.chunkProvider.saveChunks(flag, iprogressupdate);
+             } // PandaSpigot
+             // CraftBukkit - ArrayList -> Collection
++            /* // PandaSpigot start - Disable Vanilla Chunk GC
```

Comment:
maybe keep comment and comment block seperate lines

like:

```java
// PandaSpigot start - Disable Vanilla Chunk GC
/*
```

```java
*/
// PandaSpigot end
```

makes the diff a bit larger but also perhaps more readable if there are more changes around it in future